index.html

```diff
@@ -388,8 +388,8 @@ <h3 class="title chapter-title"><a id="chapter4">The Origins of Chalga</a></h3>
 						<li> (I am king and master of the party)</li>
 					</ul>
 					<figure class="videoWrapper">
-						<iframe src="https://www.youtube.com/embed/iYfER-32DZg" loading="lazy"
-							allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen>
+						<iframe src="https://www.youtube.com/embed/dITVI0ytRNE" loading="lazy"
```

Comment:
This video is still broken? Perhaps by the time I review it, it had already changed... YouTube moves on quickly :D 
Would you change it to this one `OVTxI5KxrRw`?

Reply:
For me this link works without any problems, can you check again?
https://www.youtube.com/embed/dITVI0ytRNE

Reply:
Mhm, you are correct. It seems like the embed version doesn't work. 
Regardless, it needs to have the original video clip as part of the video. Not only the music.

Reply:
Ok, I will try to find a video that haves the original video clip.

Reply:
This one has it `OVTxI5KxrRw` but it seems like the problem comes when they are embedded
`https://www.youtube.com/embed/OVTxI5KxrRw` doesn't load.
This is apparently due to some YouTube videos being restricted and you cannot embed them... I need to find a video that is satisfying both requirements 
- [ ] has the original clip
- [ ] can be embedded